pages/04.Reference/13.identity-and-access/docs.en.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Identity and Access
+published: true
+date: '20-03-2020 10:40'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
+
+OpenStack's Identity API is implemented via Keystone. It provides API client authentication, service discovery, and distributed multi-tenant authorization.
```

Comment:
distributed multi-tenant authorization does not mean anything for our clients, so I would remove that sentence.

Reply:
Alright, I will leave this one out